extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -162,11 +155,15 @@ export class AmpAdXOriginIframeHandler {
           .then(() => this.noContent_());
     }
 
+    if (opt_isA4A) {
```

Comment:
Why did you move this down here?  Does it depend on the listeners having already been set up?  If so, should probably add a note about that dependency.  Also, seems worth keeping the original comment -- I don't think it's deprecated. (?)